packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -819,6 +825,19 @@ export class SolidParser {
             const babylonMesh = new Mesh(this._meshesFromObj[j].name, scene);
             babylonMesh._parentContainer = assetContainer;
             scene._blockEntityCollection = false;
+            this._handledMesh._babylonMesh = babylonMesh;
+            // If this is a group mesh, it should have an object mesh as a parent. So look for the first object mesh that appears before it.
+            if (!this._handledMesh.isObject) {
+                for (let k = j - 1; k >= 0; --k) {
+                    if (this._meshesFromObj[k].isObject && this._meshesFromObj[k]._babylonMesh) {
+                        babylonMesh.parent = this._meshesFromObj[k]._babylonMesh!;
+                        break;
+                    }
+                    if (k <= 0) {
+                        Tools.Warn("OBJ Loader: Group mesh has no object parent.");

```

Comment:
Use logger instead of tools to not import extra uneeded toools.

Reply:
I am also not sure this should be warned. A group is a mesh. If inside an object, it should be its child, if not - the mesh has no parent.